src/main/scala/karazin/scala/users/group/week5/topic/program.scala

```diff
@@ -9,4 +9,4 @@ object program:
     else right
 
 
-
+  max(1,2)
```

Comment:
Leftovers?

Reply:
We need to write assert in program or in tests?

Reply:
Generally speaking, you shouldn't print anything in the code. It's a bad style. You should use logs for that.
Since I do not see any tests then assert is preferable than printing.